src/AppInstallerCLICore/Workflows/PromptFlow.cpp

```diff
@@ -5,13 +5,29 @@
 #include "ShowFlow.h"
 #include <winget/UserSettings.h>
 
+using namespace AppInstaller::CLI::Execution;
 using namespace AppInstaller::Settings;
 using namespace AppInstaller::Utility::literals;
 
 namespace AppInstaller::CLI::Workflow
 {
     namespace
     {
+        bool IsInteractivityAllowed(Execution::Context& context, Reporter::Level reporterLevel) {
```

Comment:
```
bool IsPromptAllowed(Execution::Context& context, Reporter::Level reporterLevel)
{
    return IsInteractivityAllowed(context) && context.Reporter.IsLevelEnabled(reporterLevel);
}
```

Use a different method name to more precisely describe the function. IsInteractivityAllowed is more of DisableInteractivity flag focused. Also, all reporter level checking should be performed within Reporter, it's not the caller's responsibility to interprete internal Reporter flags.